.drone.yml

```
@@ -198,77 +195,58 @@ trigger:
       - refs/tags/**
 
 steps:
-  - name: init
-    image: quay.io/sighup/e2e-testing-drone-plugin:v1.26.4
+  - name: create-kind-cluster
+    image: quay.io/sighup/dind-kind-kubectl-kustomize:0.20.0_1.29.1_3.10.0
     pull: always
     volumes:
+      - name: dockersock
+        path: /var/run/docker.sock
       - name: shared
         path: /shared
     depends_on: [clone]
-    settings:
-      action: custom-cluster-126
-      cluster_version: "1.26.4"
-      local_kind_config_path: katalog/tests/kind-config.yml
-      pipeline_id: cluster-126
-      instance_path: /shared
-      aws_default_region:
-        from_secret: aws_region
-      aws_access_key_id:
-        from_secret: aws_access_key_id
-      aws_secret_access_key:
-        from_secret: aws_secret_access_key
-      terraform_tf_states_bucket_name:
-        from_secret: terraform_tf_states_bucket_name
-      vsphere_server:
-        from_secret: vsphere_server
-      vsphere_password:
-        from_secret: vsphere_password
-      vsphere_user:
-        from_secret: vsphere_user
-      dockerhub_username:
-        from_secret: dockerhub_username
-      dockerhub_password:
-        from_secret: dockerhub_password
+    environment:
+      CLUSTER_VERSION: v1.27.3
+      CLUSTER_NAME: ${DRONE_REPO_NAME}-${DRONE_BUILD_NUMBER}
+      # /drone/src is the default workdir for the pipeline
+      # using this folder we don't need to mount another
+      # shared volume between the steps
+      KUBECONFIG: /drone/src/kubeconfig
```

Add some unique identifier to the kubeconfig path, otherwise it will be overwritten by the other pipelines that run in parallel for the other kubernetes versions.
```suggestion
      KUBECONFIG: /drone/src/kubeconfig-127
```